instruqt-tracks/terraform-cloud-aws-v2/03-centralize-deployments/check-workstation

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -l
+
+# Create /tmp/skip-check to disable this check
+if [ -f /tmp/skip-check ]; then
+    rm /tmp/skip-check
+    exit 0
+fi
+
+cd /root/hashicat-aws
+
+TOKEN=$(grep token /root/.terraform.d/credentials.tfrc.json | cut -d '"' -f4)
+
+# Check the token length
+[[ 91 -eq $(echo $TOKEN | wc -m) ]]
```

Comment:
Can we do some sort of call to the API and just use that as a functional token test?  I don't like trusting that any string of length 91 is correct, or that the TFC token length will never change.

Reply:
leave that for rev2